Code/EnginePlugins/OpenXRPlugin/init.cmake

```diff
@@ -0,0 +1,44 @@
+######################################
+### OpenXR support
+######################################
+
+set (EZ_BUILD_OPENXR OFF CACHE BOOL "Whether support for OpenXR should be added")
+
+######################################
+### ez_requires_openxr()
+######################################
+
+macro(ez_requires_openxr)
+
+	ez_requires_windows()
+	ez_requires(EZ_BUILD_OPENXR)
+	# While counter-intuitive, we need to find the package here so that the PUBLIC inherited
+	# target_sources using generator expressions can be resolved in the dependant projects.
+	find_package(ezOpenXR REQUIRED)
+
+endmacro()
+
+######################################
+### ez_link_target_openxr(<target>)
+######################################
+
+function(ez_link_target_openxr TARGET_NAME)
+
+	ez_requires_openxr()
+
+	find_package(ezOpenXR REQUIRED)
+
+	if (EZOPENXR_FOUND)
+		target_link_libraries(${TARGET_NAME} PRIVATE ezOpenXR::Loader)
+
+		# TODO: How to check whether the target even has a dll, e.g. is a dynamic build?
```

Comment:
IIRC there is a property on the target that tells you whether it is a SHARED library.

Reply:
https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#properties-on-targets

Not sure which property, but I think I've used something like that before.

Reply:
Actually used this in another place already to do the deployment of the dlls. This works:
get_target_property(_dll_location ezOpenXR::Loader IMPORTED_LOCATION)